### PR TITLE
"input" is a GLSL keyword

### PR DESCRIPTION
**Pull request by HdC-adsk**
Using "input" as variable name would cause shader errors on some platforms.

### CODE REVIEW

**Comment by BigRoy** [comment]
It seems like an overly verbose refactored name to make clear why it's not just `input` but admittedly I think this might be one of those things that will end up look even more confusing if you don't know about it. It's trying to make the variable name into a comment.

Would `input_` or `input_value` be better? Or even just `value`?

File: lib/render/vp2ShaderFragments/Float4ToFloatW.xml [hunk 1]
```diff
@@ -15,77 +14,34 @@
...
-            <source>
-                <![CDATA[
-// Check whether we're in a GLSL context. If so, we don't expect "floatN"
-// types to exist, so we use "vecN" types instead.
-#if defined(__VERSION__) && (__VERSION__ >= 110)
-#define float4 vec4
-#endif
-
-float
-Float4ToFloatW(float4 input)
```
Comment (BigRoy) [inline lib/render/vp2ShaderFragments/Float4ToFloatW.xml]: This PR seems to describe solely refactoring variable name `input` but it also seems to remove quite some lines of code in multiple files. Is that related to the fix? If so, how? And if not, what is the reason for removing it?

**Comment by HdC-adsk** [comment]
@BigRoy Yes, I agree the name is verbose, but on the other hand, this name is also used in similar fragments in VP2 built-in shader fragments and some plugins. So I keep using the same name here to make them consistent.

Do I skip some renaming in other shaders? If you mean HLSL and Cg, they don't reserve "input" as keyword.

Sorry to make it confusing to sneak in other unrelated change. They are removed because they are not needed by VP2 and I am trying to clean them up while I am fixing issues. The VP2RenderDelegate code is still in flux, so I don't strictly do minimum code commit when working for each bug and feature.
